integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/environment/AzureEnvironmentWithCustomerManagedKeyTests.java

```diff
@@ -74,7 +74,7 @@ protected void setupTest(TestContext testContext) {
     }
 
     @AfterMethod(onlyForGroups = { "withrg" })
-    public void tearDown(Object[] data) {
+    public void singleResourceGroupTearDown(Object[] data) {
         LOGGER.info("Tear down context");
         ((TestContext) data[0]).cleanupTestContext();
```

Comment:
in this case cleanup will triggered twice, since parent class also calling this

Reply:
so cleanupTestContext call should be removed from this method

Reply:
This is required to remove the resource group successfully at Azure portal:
- First step we are terminating all the running resources in this resources group
- then remove the resource group itself. 
 
I'm checking if the second context teardown will be invoked and the time that is consumed. I think this has been done in few seconds.

Reply:
The second teardown has been done 1 second. However this was resulting in configuration failure. So need to stop the teardown after this for this test case context.

Reply:
you can keep the method, only the cleanupTestContext in it should be removed, since it is alresdy called in another method